Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestManager.java

```diff
@@ -1398,10 +1398,6 @@ private void addPendingJob(Manifest manifest, AutoIngestJobNodeData nodeData) th
             AutoIngestJob job;
             if (nodeData.getVersion() == AutoIngestJobNodeData.getCurrentVersion()) {
                 job = new AutoIngestJob(nodeData);
-                Path caseDirectory = PathUtils.findCaseDirectory(rootOutputDirectory, manifest.getCaseName());
```

Comment:
For some unknown reason this code *really* wanted to make sure the case directory path was set on the job, so much so that there are 3 identical chunks of code to accomplish it (in the body of the 'if', in the body of the 'else' and immediately after the 'if-else'). This change eliminates 2 of them. Given that you have also made a change to openCase() to update the node data after the directory is set again, do we even need to keep the 3rd version of the code here? If there isn't a really good reason I would vote to eliminate it to avoid future confusion.

Reply:
I really wanted to keep the number of code changes to a minimum so I decided not to touch the old ZK node support code. Perhaps we should delete that all together. But sure, I can modify that code as well.  I also didn't modify the doRecoveryIfCrashed() method which calls PathUtils.findCaseDirectory(), simply because that code gets executed very rarely so who cares that we will scan the output directory when doing a job recovery.